docs/en/week12/12-1.md

```diff
@@ -0,0 +1,220 @@
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-1/figure9.png" style="background-color:#DCDCDC;"><br>
+<b>Figure 9:</b> Translation quality degrades rapidly for low resource languages
+</center>
+
+
+### Machine Translation in Practice
+Let's consider an English to Nepali Machine Translation system where we translate English news to Nepali. Nepali is a low resource language and the amount of parallel data for training is very small. 
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-1/figure10.png" style="background-color:#DCDCDC;"><br>
+<b>Figure 10:</b> Open Parallel Corpus Dataset resources for English to Nepali
+</center>
+
+The open parallel corpus is a place where we can get these datasets. There are 2 issues here:
+1. Now, if we look for Nepali dataset, we see that either there is very less amount of quality data set or huge dataset with no useful content.
+2. There are possibilities that the source and target data are not parallel. We might have more data in English in some categories, more data in Nepali in other categories. There are cases these two don't match.
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-1/figure11.png" style="background-color:#DCDCDC;"><br>
+<b>Figure 11:</b> Machine Translation from English to Nepali with the help of other languages like Hindi
+</center>
+
+One way to solve these issues is to make use of other languages related to Nepali. For example, Hindi is a much higher resource language and belongs to the same family as Nepali. We can extend this to other languages also.
+
+### Low Resource Machine Translation
+
+In practice, whenever a model is being trained, there is a lot of effort that goes in analyzing the model and the properties of the data. This whole thing is an iterative process and the machine learning practitioner needs to view this with a complete picture. Secondly, if there is less data, we can downscale the model, which isn't desirable. We need to come up with ways to enlarge the dataset somehow or use unsupervised techniques.
+
+| <center><img src="{{site.baseurl}}/images/week12/12-1/figure12.png"/></center> | <center><img src="{{site.baseurl}}/images/week12/12-1/figure13.png"/></center> |
+|                              (a) Data collection in ML                         |                              (b) Low-resource tasks in ML                      |
+
+<center>
+<b>Figures 12, 13:</b> Data collection and Low-resource tasks in ML
+</center>
+
+#### Challenges
+
+Figure 14 depicts some of the challenges encountered in Low Resource Machine Translation tasks.
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-1/figure14.png" style="background-color:#DCDCDC;"><br>
```

Comment:
This is text. Please, use text.

Reply:
I don't understand, the text is not enclosed in `$$`. What needs to be done?

Reply:
The figure is not an image, it's text.
Please, type out the text.